addons/ailogistics/functions/fnc_createPickets.sqf

```diff
@@ -0,0 +1,37 @@
+#include "script_component.hpp"
+/*
+    Author:
+        Bridg
+
+    Description:
+        Creates the picket groups
+        _picketLocations is the modules, not their positions
+
+    Parameters:
+        0: _picketLocations <ARRAY>
+
+    Return value:
+        Nothing
+*/
+
+
+if (!isServer) exitWith {};
+
+params ["_picketLocations"];
+
+{
+    private _sideFromModule = _x getVariable [QGVAR(side), 0];
+    if (_sideFromModule == 0) then {
```

Comment:
Is it planned that this could run twice for the same modules?
If so, the types are going to be out of whack due to the reassignment of the variable.

For robustness, I would resolve the side here for use only and leave the module variable intact as a number.

e.g
```
private _sideNumber = _x getVariable [QGVAR(side), 0];
private _side = [EAST, INDEPENDENT] select _sideNumber;
```